gcp/Rakefile

```diff
@@ -0,0 +1,12 @@
+import "rakefiles/deploy.rake"
+
+if !ENV['ENV']
```

Comment:
I think Ruby people prefer `unless` to `if !`.

Reply:
Thanks, `unless` fits much better here!